pkg/controller/chaosengine/chaosengine_controller_test.go

```diff
@@ -403,59 +285,3 @@ func TestGetChaosRunnerENV(t *testing.T) {
 		})
 	}
 }
-
-func TestGetChaosMonitorENV(t *testing.T) {
```

Comment:
Change the name in 166 line: I.e `Name:      "test-monitor",`

Reply:
@chandankumar4 , its the name of chaosengine, we dont have to bother with that.

Reply:
yaa!! This name is also okay